LibreNMS/OS/Edgecos.php

```diff
@@ -75,6 +76,39 @@ public function discoverOS(Device $device): void
         $device->hardware = $data[0]['swProdName'] ?? null;
         $device->serial = $data[1]['swSerialNumber'] ?? null;
     }
+class Edgecos extends OS implements TransceiverDiscovery
```

Comment:
This isn't needed and is causing code issues.

You just need to add `TransceiverDiscovery` to the implements line closer to the top.

Have you tested this? As it stands the code would break Edgecos.